flexmeasures/ui/templates/crud/user.html

```diff
@@ -67,7 +67,7 @@ <h2> Account overview for {{ user.username }} </h2>
                                 Assets owned
                             </td>
                             <td>
-                                <a href="/assets/owned_by/{{ user.id }}">{{ user.assets | length }}</a>
```

Comment:
So why did this not work?

Reply:
Our UI calls our API, but the API is not returning the assets of the user like SQLAlchemy automatically does (with a second query). Our API returns a JSON structure, and whoever wants the assets can make a second call.